Docker/test/selenium/specs/elasticsearch/elasticsearch.js

```diff
@@ -57,7 +57,7 @@ describe( 'ElasticSearch', function () {
 
 	} );
 
-	it( 'Should be able to run ForceSearchIndex.php', function () {
+	it.skip( 'Should be able to run ForceSearchIndex.php', function () {
```

Comment:
What's the plan for this? Can they be re-enabled in some other form in a follow up... maybe by making an edit and then running the script?